lib/compile/bundles/bootstrap.js

```diff
@@ -0,0 +1,21 @@
+import most from "most";
+
+import resolveAsset from "../modules/resolve";
+import loadAst from "../modules/load-ast";
+
+export default function (compilation) {
+  return most
+    .from(compilation.opts.emit)
+    .filter(def => def.entry || def.split)
+    .map(function bootstrapBundle (bundleDef) {
```

Comment:
Hm, good point about duplicate name, although I don't think the import name matters.  I haven't decided, but have considered replacing some of the arrow function with named functions, even though they're more verbose.  This would be to improve stack traces, primarily.  Either way, I should probably standardize :)